pyfar/coordinates.py

```diff
@@ -154,39 +151,31 @@ def __init__(self, points_1=None, points_2=None, points_3=None,
     def set_cart(self, points_1, points_2, points_3,
                  convention='right', unit='met'):
```

Comment:
For cartesian coordinates, the only possible convention parameter is `'right'` and only possible unit is `'met'`, right?

Reply:
for now yes - but we might want to add something in the future and it is more consistent with the remaining domains :)